src/config/entities/configuration.ts

```diff
@@ -61,6 +61,9 @@ export default () => ({
       process.env.HTTP_CLIENT_REQUEST_TIMEOUT_MILLISECONDS ?? `${5_000}`,
     ),
   },
+  log: {
+    level: process.env.LOG_LEVEL || 'debug',
```

Comment:
What happens if `LOG_LEVEL` is a random string (e.g. `foo`)?

Reply:
In that case, the service won't crash but it wouldn't log anything 🤔 Maybe it is better to include the log level validation in [its schema](https://github.com/safe-global/safe-client-gateway-nest/blob/c4e872915755574a83a18efeee91172713dc6167/src/config/configuration.validator.ts#L3) to avoid that scenario and fail fast, what do you think?

Reply:
> In that case, the service won't crash but it wouldn't log anything 🤔 Maybe it is better to include the log level validation in [its schema](https://github.com/safe-global/safe-client-gateway-nest/blob/c4e872915755574a83a18efeee91172713dc6167/src/config/configuration.validator.ts#L3) to avoid that scenario and fail fast, what do you think?

Yes I think this is a good solution 👍 We should fail fast if the log level is set but is not recognised as a valid one.

Reply:
Validation added in [d5b53b6](https://github.com/safe-global/safe-client-gateway-nest/pull/516/commits/d5b53b6ada8810891f3f206cc89af60d0f020b83)